docs/20_faq.md

```diff
@@ -5,25 +5,122 @@ sidebar_label: FAQ
 
-**A**: You will need to edit the **network.js** file found within the WebTemplates folder. Edit the ChainId to the one for the blockchain you want to game to hit.
+### Is the SDK free to use?&#x20;
 
-**Q:**  I'm seeing the following error when I install the package. **The type or namespace name 'Newtonsoft' could not be found**
+Yes, the SDK is free to use.
 
-**A:** NewtonSoft is a dependency of the SDK and needs to be installed for the SDK to work. This could be found on the Unity Asset store [JSON.NET](https://assetstore.unity.com/packages/tools/input-management/json-net-for-unity-11347)
+### Can I make desktop games with the SDK?
 
-**Q:** I'm getting the following error when trying to import an NFT texture into the SDK. **Curl error 1: Protocol "ipfs" not supported or disabled in libcurl**
+Yes, you can make Desktop / Web and Mobile games using our SDK. You will need to create a Web3GL build to make desktop / mobile games with web3.unity.
 
-**A:** IPFS has its own protocol that isn't supported in JSON.Net in order to get around that you need to add something like the following code snippet.
+### I want to connect to other EVM-compatible chains but it only lets me connect to Ethereum. How do I do that?
+
+You will need to edit the `network.js` file found within the **WebTemplates** folder. Edit the `ChainId` to the one for the blockchain you want your game to interact with. See websites like [Chainlist](https://chainlist.org/) for a list of EVM ChainID's.
+
+### Does the SDK support other non-EVM blockchains like Solana or Algorand?
+
+Not currently. Although, the SDK is mainly compatible with Ethereum Virtual Machine (EVM) blockchains, we do have plans for supporting non-EVM chains in the future depending on community support.
+
+### The Google Play / Apple AppStore is rejecting my application, what can I do?
+**Answer #1:** This usually happens because the game has not informed the user that a MetaMask pop up will occur. You can solve this by adding a simple notification as seen below.
+
+![](assets/reqMetamask.png)
+
+**Answer #2:** Some stores may require a demo version of the game so that the stores teams can review the product without needing an external wallet. You can simply add a demo button within the game that takes the user to a free-to-play version of your game without blockchain calls to test. This should be enough for the store to accept your games application. Happy coding!
+
+### How do I enable Torus and other crypto wallets for the SDK?
+
+You can do this by editing the WebGLTemplates -> WebGL2020x -> Web3 -> index.js file. You need to uncomment the wallet at the top to allow the UNPKG javascript file to be used and then uncomment the wallet in the connect function below. Other wallets can be added the same way, provided the developers of the wallet supply a UNPKG javascript file and the wallet is EVM-compatible (such as MetaMask).
+
+![](assets/EnablingAlternateWallets.png)
+
+## Common Questions About web3.unity v2
```

Comment:
I recall @alexvotofuture wishing to not want to differentiate v2 from the SDK, that they were synonymous. Has this since changed, was this misunderstood, or something else perhaps?

These questions will be top-of-mind for users, but how we frame these questions so that they don't specifically segregate v2 from being just the next version of the SDK.. this may need some additional massaging of the messaging.

Reply:
Great point. Maybe "latest update"?